shim.js

```diff
@@ -21,7 +19,7 @@ module.exports = function shimName() {
 			configurable: true,
 			enumerable: false,
 			get: function () {
-				var name = fnCall(polyfill, this);
+				var name = polyfill.bind(this).call(this);
```

Comment:
@ljharb 
IE11:
<img width="917" alt="screen shot 2017-07-10 at 19 39 23" src="https://user-images.githubusercontent.com/1754430/28048719-8ceed018-65a7-11e7-9fcb-f3ae33e4f93e.png">

```js
const bind = require('function-bind')
bind.call(polyfill, this).call(this)
```

I think this would work, will test on browserstack.